addons/docs/src/blocks/Story.tsx

```diff
@@ -66,7 +70,15 @@ export const getStoryProps = (
 const StoryContainer: React.FunctionComponent<StoryProps> = props => (
   <DocsContext.Consumer>
     {context => {
+      const { parameters } = context;
+      const disable = parameters && parameters.docs && parameters.docs.disable;
+      if (disable) {
+        return null;
+      }
       const storyProps = getStoryProps(props, context);
+      if (!storyProps) {
```

Comment:
When could this be falsey but not line 75 above?

Reply:
😭

Reply:
forgot to delete the old code